ui/block/BlockDetails.tsx

```diff
@@ -308,8 +308,9 @@ const BlockDetails = ({ query }: Props) => {
       ) }
       { !config.UI.views.block.hiddenFields?.burnt_fees && (
         <DetailsInfoItem
-          title="Burnt fees"
+          title="Reward pool fees"
           hint={
+            //DOUBLE CHECK: Update this text
```

Comment:
The tooltip copy can be the same as in the current explorer `ZEN awarded to the Reward Pool from transactions included in the block (Base fee (per unit of gas) * Gas Used).`